src/qibolab/qubits.py

```diff
@@ -57,7 +57,8 @@ class Qubit:
     g: float = 0.0
     """Readout coupling"""
 
-    assigment_fidelity: float = 0.0
+    assignment_fidelity: float = 0.0
+    fidelity: float = 0.0
```

Comment:
Maybe it's better to use something like
```suggestion
    measurement_fidelity: float = 0.0
```
to distinguish from potential gate fidelities, etc. that maybe we can calculate in the future?

Reply:
Indeed I also think that `fidelity` is a bit too generic.
Is there a specific term for the fidelity obtained using single shot classification, @Edoardo-Pedicillo @Jacfomg ?
In our case the fidelity is given by this
https://github.com/qiboteam/qibocal/blob/3bd6404e6c9ab35e725d219a6630bb6bd8a9794a/src/qibocal/fitting/classifier/qubit_fit.py#L105
Maybe we can call it `readout_fidelity`?

Reply:
Yup, it's called that if I am not mistaken.

Reply:
Great, then let's go with `readout_fidelity`. @andrea-pasquale feel free to merge after you make the change.

Reply:
Thanks for the feedback. I will merge now.